code/modules/mob/living/simple_animal/hostile/megafauna/colossus.dm

```diff
@@ -317,9 +318,9 @@
 	ActivationReaction(null, ACTIVATE_BOMB)
 	return TRUE
 
-/obj/machinery/anomalous_crystal/honk //Strips and equips you as a clown. I apologize for nothing
-	observer_desc = "This crystal strips and equips its targets as clowns."
-	possible_methods = list(ACTIVATE_MOB_BUMP, ACTIVATE_SPEECH)
+/obj/machinery/anomalous_crystal/honk //Revives the dead, but strips and equips them as a clown
+	observer_desc = "This crystal revives targets around it as clowns Oh, that's horrible...."
```

Comment:
```suggestion
	observer_desc = "This crystal revives targets around it as clowns."
```

i don't really know what you meant here

Reply:
That becoming a clown is a fate worse than death, I guess? I just missed a full stop.

Reply:
fine then, just lacked grammatical sense